Library/Homebrew/dev-cmd/tap-new.rb

```diff
@@ -127,10 +131,10 @@ def run
                 - name: Pull bottles
                   env:
                     HOMEBREW_GITHUB_API_TOKEN: ${{ github.token }}
-                    HOMEBREW_GITHUB_PACKAGES_TOKEN: ${{ github.token }}
-                    HOMEBREW_GITHUB_PACKAGES_USER: ${{ github.actor }}
+                    HOMEBREW_GITHUB_PACKAGES_TOKEN: #{args.github_packages? ? "${{ github.token }}" : "null"}
+                    HOMEBREW_GITHUB_PACKAGES_USER: #{args.github_packages? ? "${{ github.repository_owner }}" : "null"}
```

Comment:
What's the advantage in using `null` here?

Reply:
Sort of similar to how fine-grained tokens are configured, I think it's preferred not to set an environment variable when it's not needed. Though, technically this should already have been restricted via the `permissions` above with `packages: none`.

Reply:
> This is skill going to set the variable but just to a `null` value which seems weird.

It's actually null really, because it's interpreted as a YAML `null`.

> Nicer to make the output of both of these a single conditional perhaps?

ea37708591d3582e6a0857f582d9cc4290ade227 should hopefully make it better by not setting those variables at all.